app/views/find/courses/financial_support/_salaried.html.erb

```diff
@@ -3,6 +3,15 @@
     Financial support is not available for this course because it comes with a salary.
   </p>
   <p class="govuk-body">
-    You may be eligible for financial support if you choose a course without a salary. This support includes bursaries, scholarships and loans.
+    <a href="https://www.gov.uk/government/publications/national-pay-scales-for-eligible-teaching-and-education-jobs/national-pay-scales-for-eligible-teaching-and-education-leadership-occupation-codes">Find out more about national pay scales for unqualified teachers</a>
+  </p>
+  <p class="govuk-body">
+    Salaried courses are in high demand, so check your eligibility with the training provider and apply as soon as possible.
+  <p>
+  <p class="govuk-body">
+    If you choose a course without a salary, you may be eligible for financial support. This includes bursaries, scholarships and student loans.
+  </p>
+  <p class="govuk-body">
+    <a href="https://getintoteaching.education.gov.uk/funding-and-support">Learn more about ways to fund your training</a>
```

Comment:
Can we tuck these links into the translations? 

This get into teaching one can probably go here, following a similar pattern: https://github.com/DFE-Digital/publish-teacher-training/blob/a9237566dbb2857841b0f934ac0300ca09c097c9/config/locales/find.yml#L118-L131

For the publications one, perhaps we could make a links section in `find.yml` similar to: https://github.com/DFE-Digital/publish-teacher-training/blob/a9237566dbb2857841b0f934ac0300ca09c097c9/config/locales/en.yml#L889-L900

Reply:
Yes that's fine. Can you let me review again when done please?